apps/nextjs-website/src/_contents/makeDocs.ts

```diff
@@ -26,6 +27,11 @@ export type GuideDefinition = {
   readonly bannerLinks: readonly BannerLinkProps[];
 };
 
+export type SolutionDefinition = {
```

Comment:
Why do you need a new type instead of using Solution from `apps/nextjs-website/src/lib/types/solution.ts`?